modules/ServerAPI/src/client/BaseEntity.coffee

```diff
@@ -153,7 +153,9 @@ class BaseEntity extends Backbone.Model
 				saveNotebook= window.conf.entity.notebook.save
 			requireNotebook = true #default
 			if window.conf.entity?.notebook?.require?
-				requireNotebook= window.conf.entity.notebook.require
+				requireNotebook = window.conf.entity.notebook.require
+				if window.conf[attrs.subclass]?.notebook?.require?
```

Comment:
This makes sense. `client.entity.notebook.require` is the default setting for all entities, then if `client.protocol.notebook.require` or `client.experiment.notebook.require` are set, they override the default.